src/main/java/edu/ucsb/ucsbcslas/models/TutorAssignmentModel.java

```diff
@@ -0,0 +1,165 @@
+package edu.ucsb.ucsbcslas.models;
+
+import java.util.Objects;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import com.opencsv.bean.CsvBindByName;
+import com.opencsv.bean.CsvBindByPosition;
+import org.apache.commons.lang3.builder.EqualsBuilder;
+
+public class TutorAssignmentModel {
+
+    @Column(nullable = false)
```

Comment:
Since we are not directly storing this class in a repository, we can remove all Column annotations

Reply:
Done